src/Test/Utilities/Portable/Compilation/TestOperationVisitor.cs

```diff
@@ -44,26 +44,17 @@ public override void Visit(IOperation operation)
                 var type = operation.Type;
                 var constantValue = operation.ConstantValue;
 
-                // operation.Language can throw due to https://github.com/dotnet/roslyn/issues/23819
```

Comment:
>https://github.com/dotnet/roslyn/issues/23819 [](start = 55, length = 45)

Would it make sense to add a unit-test asserting IOperation tree for the scenario described in the issue? Or do we have one already?

Reply:
@333fred If you are not planning to add such a unit test, I can do it as part of https://github.com/dotnet/roslyn/issues/23819.

Reply:
Ah, never mind, seems that issue is assigned to Fred as well. I will let you decide which PR to add test.

Reply:
This is the PR to fix that issue :)
I'll add a test to this PR.

---
In reply to: [197884158](https://github.com/dotnet/roslyn/pull/28084#discussion_r197884158) [](ancestors = 197884158)